### PR TITLE
DM-33326: Fix round-tripping of spectractor spectra

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: spectractor/extractor/spectrum.py [hunk 1]
```diff
@@ -553,14 +590,20 @@ def save_spectrogram(self, output_file_name, overwrite=False):
         hdu.writeto(output_file_name, overwrite=overwrite)
         self.my_logger.info('\n\tSpectrogram saved in %s' % output_file_name)
 
-    def load_spectrum(self, input_file_name):
+    def load_spectrum(self, input_file_name, spectrogram_file_name_override, psf_file_name_override):
```
Comment (czwa) [inline spectractor/extractor/spectrum.py]: Are the new arguments mandatory?  They're optional in the `__init__` method.

File: spectractor/extractor/targets.py [hunk 2]
```diff
@@ -275,7 +275,9 @@ def load_spectra(self):
                 else:
                     self.wavelengths.append(data.wave)
                     self.spectra.append(data.flux)
-        elif 'HD' in self.label or self.label in parameters.STAR_NAMES:  # it is a star
+        # TODO: the use of self.label in parameters.STAR_NAMES:
```
Comment (czwa) [inline spectractor/extractor/targets.py]: Ticket reference for this fix?

**Review by czwa — APPROVED** [review]
Is there a unit test to catch round-tripping issues?

File: spectractor/extractor/spectrum.py (hunk 1 above)
Comment (mfisherlevine) [reply spectractor/extractor/spectrum.py]: Nope, shouldn't be, that's a great catch, thanks!

**Comment by mfisherlevine** [comment]
Not yet for the round-tripping question (hence it all being so broken in the first place!). I think we need a CI package for atmospec+Spectractor tbh, but I think that's out of scope for now. Maybe we can discuss on Monday.

**Pull request merged**